doc/instances.md

```diff
@@ -847,6 +847,18 @@ gid         | int       | 0                 | no        | GID of the device owne
 mode        | int       | 0660              | no        | Mode of the device in the instance
 required    | boolean   | false             | no        | Whether or not this device is required to start the instance. (The default is false, and all devices are hot-pluggable)
 
+
+### Type: tpm
+
+Supported instance types: container, VM
+
+TPM device entries enable access to a TPM emulator.
+
+The following properties exist:
+
+Key         | Type      | Default           | Required  | Description
+required    | boolean   | true              | no        | Whether or not this device is required to start the instance
```

Comment:
Given that this is a vTPM that we create and not something that we pass through from the host, I don't think that `required` makes sense.

If a user adds a `tpm` device to their instance and the emulator can't be started, we should just fail startup entirely.

So for now, it looks like this will be a device type without any configuration (a first!).